addons/common/field.jsx

```diff
@@ -4,27 +4,42 @@
  */
 
 let React = require('react')
+let uid   = 0
 
-module.exports = React.createClass({
+let Field = React.createClass({
 
   getDefaultProps() {
     return {
+      hint    : null,
       element : 'input',
       type    : 'text'
     }
   },
 
+  getInitialState() {
+    return {
+      hintId: `hint-col-field-${uid++}`
```

Comment:
seems like it could be nicer to use a more global uuid assignment mechanism for things like this.